test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAHelloWorldProject.cs

```diff
@@ -175,6 +175,80 @@ public static void Main()
                 .And
                 .HaveStdOutContaining("Hello from a netcoreapp2.0.!");
         }
+        
+        //Note: Pre Netcoreapp2.0 standalone activation uses renamed dotnet.exe
+        //      While Post 2.0 we are shifting to using apphost.exe, so both publish needs to be validated
+        [Theory]
+        [InlineData("win-arm")]
+        [InlineData("win8-arm")]
+        [InlineData("win81-arm")]
+        [InlineData("win10-arm")]
+        public void Publish_standalone_post_netcoreapp2_arm_app(string runtimeIdentifier)
+        {
+            // Tests for existence of expected files when publishing an ARM project
+            // See https://github.com/dotnet/sdk/issues/1239
+
+            if (UsingFullFrameworkMSBuild)
+            {
+                //  Disabled on full framework MSBuild until CI machines have VS with bundled .NET Core / .NET Standard versions
+                //  See https://github.com/dotnet/sdk/issues/1077
+                return;
+            }
+
+            var targetFramework = "netcoreapp2.0";
+
+            TestProject testProject = new TestProject()
+            {
+                Name = "Hello",
+                IsSdkProject = true,
+                TargetFrameworks = targetFramework,
+                RuntimeIdentifier = runtimeIdentifier,
+                IsExe = true,
+            };
+            
+            testProject.SourceFiles["Program.cs"] = @"
+using System;
+public static class Program
+{
+    public static void Main()
+    {
+        Console.WriteLine(""Hello from an arm netcoreapp2.0.!"");
+    }
+}
+";
+            var testProjectInstance = _testAssetsManager.CreateTestProject(testProject);
+
+            testProjectInstance.Restore(Log, testProject.Name);
+            var publishCommand = new PublishCommand(Log, Path.Combine(testProjectInstance.TestRoot, testProject.Name));
+            var publishResult = publishCommand.Execute();
+
+            publishResult.Should().Pass();
+
+            var publishDirectory = publishCommand.GetOutputDirectory(
+                targetFramework: targetFramework,
+                runtimeIdentifier: runtimeIdentifier);
+            
+            // The name of the self contained executable depends on the runtime identifier.
+            // For Windows family ARM publishing, it'll always be Hello.exe.
+            // We shouldn't use "Constants.ExeSuffix" for the suffix here because that changes
+            // depending on the RuntimeInformation
+            var selfContainedExecutable = "Hello.exe";
+            
+            string selfContainedExecutableFullPath = Path.Combine(publishDirectory.FullName, selfContainedExecutable);
```

Comment:
Yes - sorry, I only noticed you asked this earlier, for some reason some comments are collapsed and tagged as outdated, I missed them. I'll update this when I'm in front of my dev machine to remove the unused variable and address any earlier comments I missed.